File: app/src/main/java/ir/amozkade/hamed/datetimepicker/TestActivity.java
package ir.amozkade.hamed.datetimepicker;

import android.os.Bundle;

import java.sql.Timestamp;

import androidx.appcompat.app.AppCompatActivity;
import androidx.databinding.DataBindingUtil;
import ir.amozkade.hamed.datetimepicker.databinding.ActivityTestBinding;
import ir.amozkade.hamed.datetimepickerlib.mvvm.view.DateListener;

public class TestActivity extends AppCompatActivity implements DateListener {

    private ActivityTestBinding mBinding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mBinding = DataBindingUtil.setContentView(this, R.layout.activity_test);
        mBinding.setHandler(this);
        mBinding.dtTest.listener = this;

        mBinding.dtTest.setDateTime(null, null);
//        mBinding.dtTest.setDateTime("1396/12/30", "09:10");

    }

    public void getDateTaped() {
        mBinding.dtTest.getDateTime().getDate();
    }

    @Override
    public void onDateChange(String date, String time, Timestamp timestamp) {

    }
}

File: app/src/main/java/ir/amozkade/hamed/datetimepicker/TestActivity.kt
package ir.amozkade.hamed.datetimepicker

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View

import ir.amozkade.hamed.datetimepickerlib.DateTime

class TestActivity : AppCompatActivity() {
    lateinit var dtTest: DateTime
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_test_dialog)
        dtTest = findViewById(R.id.dtTest) as DateTime
        //        dtTest.setDate("96/11/30");
        //        dtTest.setTime("19:42");
        dtTest.setText()
        var a ="test"
        var b ="a value is $a"
    }

    fun click(view: View) {
        for (i in 1..10){
        }
        dtTest.date
        dtTest.time
    }
}
